keps/NNNN-kep-template/README.md

```diff
@@ -452,6 +453,36 @@ You can take a look at one potential example of such test in:
 https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
 -->
 
+### Testing Quality
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### What is the current (unit) test coverage of packages that will be touched when implementing this KEP?
+
+<!--
+List the packages where non-boilerplate code will be added/changed with their
+current coverage as reported by test-coverage suite under
+https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit
+-->
+
+###### If the coverage is below 80%, what additional tests will be added before implementing this KEP?
+
+<!--
```

Comment:
Maybe ask instead: does the sig have a plan to increase existing test coverage? (add a link to any related items).

Increasing test coverage isn't the responsibility of 1 author, it's the responsibility of the entire sig.

Reply:
> Increasing test coverage isn't the responsibility of 1 author, it's the responsibility of the entire sig.

Agree with this - I will rephrase inline with the comments above.